src/api/app/components/watchlist_component.rb

```diff
@@ -11,16 +11,36 @@ class WatchlistComponent < ApplicationComponent
     'BsRequest' => 'Watch this request'
   }.freeze
 
-  def initialize(user:, project: nil, package: nil, bs_request: nil)
+  def initialize(user:, bs_request: nil, package: nil, project: nil)
```

Comment:
What is the reason behind this change?

Reply:
To arrange the parameters in the same order they will be called inside the initialize, a couple of lines below:
```
@object_to_be_watched = object_to_be_watched(bs_request, package, project)
```

Reply:
Humm... alright.